lib/ansible/plugins/lookup/hashi_vault.py

```diff
@@ -181,10 +181,11 @@ def auth_ldap(self, **kwargs):
             raise AnsibleError("Authentication method ldap requires a password")
 
         mount_point = kwargs.get('mount_point')
-        if mount_point is None:
-            mount_point = 'ldap'
 
-        self.client.auth_ldap(username, password, mount_point)
+        if not mount_point:
+            self.client.auth_ldap(username, password)
```

Comment:
Is this going to alter the default behavior of the module or is the `mount_point = 'ldap'` inferred by `client.auth_ldap` ?

Reply:
The default mount_point will stay `ldap`.
This module is using [`hvac`](https://github.com/hvac/hvac) as library and `ldap` is the default mount point if not specified:

```python
            DEFAULT_MOUNT_POINT = 'ldap'
...
            def login(self, username, password, use_token=True, mount_point=DEFAULT_MOUNT_POINT):
```

https://github.com/hvac/hvac/blob/5b42620ddac5474574eb98437d2645dade1824c1/hvac/api/auth_methods/ldap.py#L8
https://github.com/hvac/hvac/blob/5b42620ddac5474574eb98437d2645dade1824c1/hvac/api/auth_methods/ldap.py#L355